_ont/ont-huawei-ma5671a-fs-mod.md

```diff
@@ -0,0 +1,188 @@
+```
+
+Once connected via telnet to the SFP, execute the following commands, it will be necessary to wait a few minutes until the end of the automatic procedure:
+
+```sh
+firstboot
+reboot
+```
+
+Remember that with each flash of this firmware it is always necessary to perform the procedure described above, otherwise the SFP will not work!
+
+# Login info
+|                  |                                                |
+| ---------------- | ---------------------------------------------- |
+| Username         | root                                           |
+| Password         | root                                           |
+| IP address	     | 192.168.1.10                                   |
+| Telnet    	     | Always available                               |
+| TTL Serial 	     | Always available                               |
+| SSH       	     | Only after the first configuration (firstboot) |
+| Web       	     | NO                                             |
+
+# GPON ONU status
+
+## Get the operational status of the ONU
+```sh
+onu ploam_state_get
+```
+
+## Get optical laser status
+```sh
+otop -g s
+```
+
+## Get the information of the OLT
+```sh
+omci_pipe.sh meg 131 0
+```
+
+## Querying a particular OMCI ME
+```sh
+omci_pipe.sh meg MIB_IDX ME_IN
+```
+Where `MIB_IDX` is the MIB ID and the `ME_IN` is the ME instance number
+
+## Get VLAN table rule
+```sh
+gtop -g "GPE VLAN rule"
+```
+
+## Get GEM port status table
+```sh
+gtop -g e
+```
+
+## Getting/Setting Speed LAN Mode
+
+To get the LAN Mode:
+
+```sh
+onu lan_port_status_get 0
+```
+The `link_status` variable tells the current speed
+
+| Value (for `sgmii_mode` and `link_status`) | Speed                              |
+| ------------------------------------------ | ---------------------------------- |
+| 3                                          | 1 Gbps / SGMII with auto-neg on    |
+| 4                                          | 1 Gbps / SGMII with auto-neg off   |
+| 5                                          | 2.5 Gbps / HSGMII with auto-neg on |
+
+To change the default lan mode value you can use `fw_setenv sgmii_mode`. The firmware is already set to 2.5G auto-negotiation, you shouldn't touch it.
+
+# GPON/OMCI settings
+
+## Setting ONU GPON serial number
+```sh
+fw_setenv onu_serial "YOUR_SERIAL_ASCII"
+```
+
+## Setting ONU GPON ploam
+```sh
+fw_setenv onu_ploam "YOUR_PLOAM_ASCII"
+```
+
+## Setting ONU GPON loid
+```sh
+fw_setenv onu_loid "YOUR_LOID"
+```
+
+## Setting ONU GPON loid password
+```sh
+fw_setenv onu_loid "YOUR_LOID_PASSWORD"
+```
+
+## Setting OMCI vendor ID
+```sh
+fw_setenv omci_vendor_id "YOUR_VENDOR_ID"
+```
+
+## Setting OMCI equipment ID
+```sh
+fw_setenv omci_equip_id "YOUR_EQUIP_ID"
+```
+
+## Setting OMCI hardware ID
+```sh
+fw_setenv omci_hw_ver "YOUR_HW_VER"
+```
+
+## Setting OMCI software version
+```sh
+fw_setenv image0_version "YOUR_SW_VERSION_0"
+fw_setenv image1_version "YOUR_SW_VERSION_1"
+```
+
+# Advanced settings
```

Comment:
I like the division into 3 parts. I think it should also be made on other pages.